platform/cog-platform-fdo.c

```diff
@@ -1081,6 +1182,8 @@ init_wayland (GError **error)
         return FALSE;
     }
 
+    bzero(&wl_data.metrics, sizeof(output_metrics) * 16);
```

Comment:
Nowadays using `memset` is preferred over `bzero`, because it has been removed from the latest POSIX specs. Also, please use `G_N_ELEMENTS` instead of `16`.